src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -52,6 +55,7 @@ public static partial class PlatformDetection
         public static bool IsRedHatFamily => IsRedHatFamilyAndVersion();
         public static bool IsNotRedHatFamily => !IsRedHatFamily;
         public static bool IsRedHatFamily7 => IsRedHatFamilyAndVersion(7);
+        public static bool IsCentos7 => IsDistroAndVersion("centos", 7);
```

Comment:
can we use the `IsRedHatFamily7` above?

Reply:
It includes RHEL (or rather excludes from testing) as well. So no, I don't think that's advisable as we want exact list of images that are failing.

Reply:
they should be basically identical. but I guess they are not:
https://github.com/dotnet/arcade/issues/10253#issuecomment-1215320634

Perhaps something we should investigate. We have at least one of the runs in container so it should be easy to check.